.github/workflows/ci.yaml

```diff
@@ -0,0 +1,53 @@
+name: Continuous integration
+on: [pull_request, push]
+
+jobs:
+  rust-tests:
+    name: Rust tests
+    env:
+      RUSTFLAGS: -D warnings
+      CARGO_INCREMENTAL: 0
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v1
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+          components: rustfmt, rust-src
+      - name: Compile
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --tests
```

Comment:
This should be `test —no-run`, the build is only for integration tests, which we don’t ise